statsrunner/loop.py

```diff
@@ -10,10 +10,22 @@
 import statsrunner.aggregate
 from statsrunner.common import decimal_default
 
+
 def call_stats(this_stats, args):
+    """Create dictionary of enabled stats for this_stats object.
+
+    Args:
+      this_stats: stats_module class object that specifies calculations for
```

Comment:
The type can be specified after the argument name.

http://www.sphinx-doc.org/en/stable/ext/example_google.html